src/common/components/cards/how-to-fix-android-card-row.tsx

```diff
@@ -0,0 +1,82 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import * as React from 'react';
```

Comment:
should this file be under `electron` folder? maybe even `electron\views\automated-checks\components\cards`

Reply:
this will be used by the configuration property, just like how we do for web https://github.com/microsoft/accessibility-insights-web/blob/cda39b427c80b79f26fc4a40c8b165617b3c21fd/src/common/configs/unified-result-property-configurations.tsx#L43

Reply:
I saw the same while working on rendering `text`, `contentDescription` and `className` so I think this is fine until we properly decide what to do with it